iRecruit-ui/src/components/Profile/Profile.jsx

```diff
@@ -0,0 +1,175 @@
+import React, { useState, useEffect } from "react";
+import './Profile.css';
+
+function Profile() {
+  const [photo, setPhoto] = useState(null);
+  const [bio, setBio] = useState("");
+  const [education, setEducation] = useState("");
+  const [currentInstitution, setCurrentInstitution] = useState("");
+  const [skills, setSkills] = useState("");
+  const [experience, setExperience] = useState("");
+  const [linkedinUrl, setLinkedinUrl] = useState("");
+  //all of these can be updated, hence the need for state hooks
+  const [profileUpdated, setProfileUpdated] = useState(false);
+
+  useEffect(() => {
+    // Retrieve the profile information from localStorage when the component mounts
+    const storedProfile = localStorage.getItem("profile");
+    if (storedProfile) {
+      const { photo, bio, education, currentInstitution, skills, experience, linkedinUrl } = JSON.parse(storedProfile);
+      setPhoto(photo);
+      setBio(bio);
+      setEducation(education);
+      setCurrentInstitution(currentInstitution);
+      setSkills(skills);
+      setExperience(experience);
+      setLinkedinUrl(linkedinUrl);
+    }
+  }, []);
+
+  const handlePhotoChange = (event) => {
+    const file = event.target.files[0];
+    if (file) {
+      const reader = new FileReader();
+      reader.onloadend = () => {
+        setPhoto({
+          file,
+          dataURL: reader.result,
+        });
+      };
+      reader.readAsDataURL(file);
+    } else {
+      setPhoto(null);
+    }
+  };
+
+  const handleBioChange = (event) => {
+    setBio(event.target.value);
+  };
+
+  const handleEducationChange = (event) => {
+    setEducation(event.target.value);
+  };
+
+  const handleCurrentInstitutionChange = (event) => {
+    setCurrentInstitution(event.target.value);
+  };
+
+  const handleSkillsChange = (event) => {
+    setSkills(event.target.value);
+  };
+
+  const handleExperienceChange = (event) => {
+    setExperience(event.target.value);
+  };
+
+  const handleLinkedinUrlChange = (event) => {
+    setLinkedinUrl(event.target.value);
+  };
+
+  const handleUpdate = (event) => {
+    event.preventDefault();
+    // save user profile: use localStorage or a state management library like Redux (Lee's suggestion)
+
+    // Create an object to store the profile information
+    const profileData = {
+      photo,
+      bio,
+      education, currentInstitution,
+      skills,
+      experience,
+      linkedinUrl
+    };
+
+    // Save the profile information to localStorage
+    localStorage.setItem("profile", JSON.stringify(profileData));
+
+    setProfileUpdated(true);
+
+    setTimeout(() => {
+      setProfileUpdated(false);
+    }, 3500);
+    console.log("Profile information saved successfully!");
```

Comment:
For checked-in code, its best to use one of the appropriate levels of log.  In this case here it could be:

`console.debug` or `console.info`

Reply:
I have looked up both levels and what they do. I have switched to console.info instead. Thank you